src/lib/menu/README.md

```diff
@@ -114,8 +114,9 @@ Output:
 
 ### Customizing menu position
 
-By default, the menu will display after and below its trigger.  You can change this display position 
-using the `x-position` (`before | after`) and `y-position` (`above | below`) attributes.  
+By default, the menu will display after and below its trigger.  You can change this display position
+using the `x-position` (`before | after`) and `y-position` (`above | below`) attributes. The menu
+can be positioned over the menu button or outside using `overlay-trigger` (`true | false`).
```

Comment:
How about "overlap-trigger"?  "Overlay-trigger" might sound like the trigger itself is an overlay.